### PR TITLE
AGENT-990: Allow control planes with 5 and 4 replicas

### PR DESCRIPTION
**Pull request by rwsu**

### CODE REVIEW

File: pkg/asset/agent/installconfig.go [hunk 1]
```diff
@@ -214,9 +214,9 @@ func (a *OptionalInstallConfig) validateControlPlaneConfiguration(installConfig
 	var fieldPath *field.Path
 
 	if installConfig.ControlPlane != nil {
-		if *installConfig.ControlPlane.Replicas != 1 && *installConfig.ControlPlane.Replicas != 3 {
+		if *installConfig.ControlPlane.Replicas != 1 && *installConfig.ControlPlane.Replicas != 3 && *installConfig.ControlPlane.Replicas != 4 && *installConfig.ControlPlane.Replicas != 5 {
```
Comment (andfasano) [inline pkg/asset/agent/installconfig.go:217]:
```suggestion
		if *installConfig.ControlPlane.Replicas < 1 || *installConfig.ControlPlane.Replicas > 5 ||*installConfig.ControlPlane.Replicas == 2 {
```
Slightly shorter comparison

File: pkg/asset/agent/installconfig.go [hunk 2]
```diff
@@ -215,9 +215,9 @@ func (a *OptionalInstallConfig) validateControlPlaneConfiguration(installConfig
 	var fieldPath *field.Path
 
 	if installConfig.ControlPlane != nil {
-		if *installConfig.ControlPlane.Replicas != 1 && *installConfig.ControlPlane.Replicas != 3 {
+		if *installConfig.ControlPlane.Replicas < 1 || *installConfig.ControlPlane.Replicas > 5 || *installConfig.ControlPlane.Replicas == 2 {
```
Comment (andfasano) [inline pkg/asset/agent/installconfig.go:218]: (and very likely we'll have to remove the `2` case after https://github.com/openshift/installer/pull/9159 will land)

**Review by pawanpinjarkar — APPROVED** [review]
/lgtm

**Comment by rwsu** [comment]
Needed to update integration test to include userManagedNetworking: false after rebase.

**Pull request merged**